README.md

```diff
@@ -28,6 +28,12 @@ juju run-action mongodb/<unit_number> get-primary --wait
 
 Similarly, the primary replica is displayed as a status message in `juju status`, however one should note that this hook gets called on regular time intervals and the primary may be outdated if the status hook has not been called recently. 
 
+Further we highly suggest configuring the status hook to run frequently. In addition to reporting the primary, secondaries, and other statuses, the status hook performs self healing in the case of a network cut. To change the frequency of the update status hook do:
+```shell
+juju model-config update-status-hook-interval=<time(s/m/h)>
```

Comment:
Is there any downsides for doing so in production env?

Reply:
Great question. Obviously one of the drawbacks of this design (which I think you face as well) is that the healing is done only as often as the update status frequency. But AFAICT this is my (and your) only option for network self healing

Fortunately the mongod command that is run in update status is cheap and shouldn't be expensive in terms of run time so this shouldn't slow any hooks down.

However, one notable draw back is this command executing too frequently when the database is under high read requests (or update status running so quickly that it prevents other hooks from running.) I can add something to this effect to the README thanks for the idea @paulomach  :)

Reply:
Sure, this is an issue I guess everyone will face. My assumption is, with bigger deployments this kind of issue tend to dilute, in the sense that one unit being down for the standard interval won't affect the related application(s) as much. So yeah, adding some notice to the users in the README is the way to go.